software/readme.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Stellar Software
+---
+# Software
+
+Pre-built software and services you can run on your own infrastructure, provided by Stellar.org.
```

Comment:
@jessica-collier: @irisli suggested the page title and overview here could be a little better and more descriptive.